internal/app/cli/LICENSE

```diff
@@ -0,0 +1,165 @@
+                   GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
regardless of the license issue, we probably don't need a LICENSE file for each subpackage

Reply:
I assumed we will have one license for Storj nodes and another for client code (like we have now in v2). Since all code goes into a single repo, I felt the need for verbosity.